syllabus/curricula/annealing_box.py

```diff
@@ -0,0 +1,48 @@
+import typing
+from typing import Any, List, Union, Sequence, SupportsFloat
+import numpy as np
+
+from gymnasium.spaces import Box
+from syllabus.core import Curriculum
+
+class AnnealingBoxCurriculum(Curriculum):
+    REQUIRES_STEP_UPDATES = True
+    REQUIRES_CENTRAL_UPDATES = False
+
+    def __init__(
+        self,
+        *curriculum_args,
+        start_values: List[SupportsFloat],
+        end_values: List[SupportsFloat],
+        steps: int = 5,
```

Comment:
I would call this `total_steps` and remove the default value, since it will be very different depending on the environment, algorithm, etc.

Also, could we make this a list? That way we can anneal one value for the first half of training, another for the entirety of training, etc.

Reply:
yeh, I've updated this